### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: Dockerfile.mayhem [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```
Comment (xansec) [inline Dockerfile.mayhem]: Same thing here, do we have to clone twice? Can we just copy the lib from the build stage?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Dockerfile.mayhem (hunk 1 above)
Comment (rnshah9) [reply Dockerfile.mayhem]: Huh; it actually wasn't even necessary.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**